servicenow/stackstate_checks/servicenow/servicenow.py

```diff
@@ -244,37 +238,41 @@ def _get_json_batch(self, url, offset, batch_size, timeout, auth):
         limited_url = url + limit_args
         return self._get_json(limited_url, timeout, auth)
 
-    def _get_json(self, url, timeout, auth=None, verify=True):
-        tags = ["url:%s" % url]
-        msg = None
-        status = None
-        resp = None
-        try:
-            resp = requests.get(url, timeout=timeout, auth=auth, verify=verify)
-            if resp.status_code != 200:
-                status = AgentCheck.CRITICAL
-                msg = "Got %s when hitting %s" % (resp.status_code, url)
-        except requests.exceptions.Timeout:
-            # If there's a timeout
-            msg = "%s seconds timeout when hitting %s" % (timeout, url)
-            status = AgentCheck.CRITICAL
-        except Exception as e:
-            msg = str(e)
-            status = AgentCheck.CRITICAL
-        finally:
-            if status is AgentCheck.CRITICAL:
-                self.service_check(self.SERVICE_CHECK_NAME, status, tags=tags,
-                                   message=msg)
-                raise CheckException("Cannot connect to ServiceNow CMDB, please check your configuration.")
-
-        if resp.encoding is None:
-            resp.encoding = 'UTF8'
+    @staticmethod
+    def _get_json(url, timeout, auth=None, verify=True):
+        execution_time_exceeded_error_message = 'Transaction cancelled: maximum execution time exceeded'
+
+        response = requests.get(url, timeout=timeout, auth=auth, verify=verify)
```

Comment:
Why did this move out of a try, no longer catching the `requests.exceptions.Timeout`?

Reply:
We do in the main `check` method. All exception from here are handled there and they got critical service_check.
I added a test `test_get_json_timeout` that checks that.